controllers/provider-openstack/pkg/webhook/controlplane/ensurer.go

```diff
@@ -102,29 +101,49 @@ func ensureKubeControllerManagerAnnotations(t *corev1.PodTemplateSpec) {
 
 var (
 	cloudProviderConfigVolumeMount = corev1.VolumeMount{
-		Name:      openstack.CloudProviderConfigName,
+		Name:      openstack.CloudProviderConfigCloudControtrollerManagerName,
 		MountPath: "/etc/kubernetes/cloudprovider",
 	}
 	cloudProviderConfigVolume = corev1.Volume{
-		Name: openstack.CloudProviderConfigName,
+		Name: openstack.CloudProviderConfigCloudControtrollerManagerName,
 		VolumeSource: corev1.VolumeSource{
 			ConfigMap: &corev1.ConfigMapVolumeSource{
-				LocalObjectReference: corev1.LocalObjectReference{Name: openstack.CloudProviderConfigName},
+				LocalObjectReference: corev1.LocalObjectReference{Name: openstack.CloudProviderConfigCloudControtrollerManagerName},
+			},
+		},
+	}
+	cloudProviderConfigKubeControllerManagerVolumeMount = corev1.VolumeMount{
+		Name:      openstack.CloudProviderConfigKubeControllerManagerName,
+		MountPath: "/etc/kubernetes/cloudprovider",
+	}
+	cloudProviderConfigKubeControllerManagerVolume = corev1.Volume{
+		Name: openstack.CloudProviderConfigKubeControllerManagerName,
+		VolumeSource: corev1.VolumeSource{
+			ConfigMap: &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{Name: openstack.CloudProviderConfigKubeControllerManagerName},
 			},
 		},
 	}
 )
 
+func ensureKubeAPIIVolumeMounts(c *corev1.Container) {
```

Comment:
```suggestion
func ensureKubeAPIServerVolumeMounts(c *corev1.Container) {
```

But why has this function been created in the first place? It's exactly the same like `ensureVolumeMounts`. Why can't you keep it using when mutating the kube-apiserver deployment?